apps/server/src/shared/infra/identity-management/keycloak-configuration/uc/keycloak-configuration.uc.ts

```diff
@@ -17,8 +17,8 @@ export class KeycloakConfigurationUc {
 		return this.kcAdmin.testKcConnection();
 	}
 
-	public async clean(): Promise<number> {
-		return this.keycloakSeedService.clean();
+	public async clean(pagination?: number): Promise<number> {
```

Comment:
To make it more clear `pageSize` instead of `pagination`.

```suggestion
	public async clean(pageSize?: number): Promise<number> {
```